status.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
What is the solution for this issue?
Is there any why we can give access to it using BUILD file?